Client/Frontend/Theme/ThemeManager.swift

```diff
@@ -26,6 +26,17 @@ enum ThemeType: String {
             return .black
         }
     }
+
+    func getLockImage() -> UIImage {
+        let imageString: String
+        switch self {
+        case .light:
+            imageString = "lock_blocked"
+        case .dark:
+            imageString = "lock_blocked_dark"
+        }
+        return UIImage(imageLiteralResourceName: imageString)
+    }
```

Comment:
@OrlaM I wasn't sure on how to best do this, let me know if you have a better way 🤔

Reply:
This should be tied to the theme instead of the manager. You could add an extra variable to the Theme protocol called images, following a similar pattern to the colors? That might be a bit overkill for just one image, alternative option you could add this getLockImage function as an extension on the Theme protocol and replace self with self.themeType, if that grows beyond a couple of images tho it'll get kind of gross.

Reply:
It's tied in the ThemeType not the manager (wasn't super clear because of the name of the file). Yeah I agree if there's too many images this will become weird. One thing I can think of is to create a method that will add "_dark" for us when we need the dark mode icon. We would just need to input the name of the image and then it will return the right dark or light mode. This might means we'll refactor some image name at some point. What do you think?

Reply:
Yeah I like that. It would also be fine for that to live in the ThemeType.